packages/api/src/controllers/dashboard.ts

```diff
@@ -48,41 +96,10 @@ export async function deleteDashboardAndAlerts(
 export async function updateDashboard(
```

Comment:
we should write a few more integration tests for cases of updating dashboard

Reply:
added!